Keas.Core/Domain/Location.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Keas.Core.Domain
+{
+    public class Location
```

Comment:
@jcubbage i forget is a location supposed to be tied to a team or are they system-wide

Reply:
I think we said they were tied to a team...